Quadriga/src/main/java/edu/asu/spring/quadriga/service/resolver/impl/ProjectHandleResolverManager.java

```diff
@@ -43,32 +45,66 @@ public boolean saveProjectHandleResolver(IProjectHandleResolver resolver, String
             resolver.setId(id);
             resolver.setUsername(username);
         }
-        
+
         ProjectHandleResolverDTO resolverDto = mapper.mapProjectHandleResolver(resolver);
         resolverDao.saveOrUpdateDTO(resolverDto);
-        
+
         return true;
     }
-    
-    /* (non-Javadoc)
-     * @see edu.asu.spring.quadriga.service.resolver.impl.IProjectHandleResolverManager#getProjectHandleResolver(java.lang.String)
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see edu.asu.spring.quadriga.service.resolver.impl.
+     * IProjectHandleResolverManager#getProjectHandleResolver(java.lang.String)
      */
     @Override
     public IProjectHandleResolver getProjectHandleResolver(String id) {
         ProjectHandleResolverDTO dto = resolverDao.getDTO(id);
         return mapper.mapProjectHandleResolver(dto);
     }
-    
-    /* (non-Javadoc)
-     * @see edu.asu.spring.quadriga.service.resolver.impl.IProjectHandleResolverManager#getProjectHandleResolvers(java.lang.String)
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see edu.asu.spring.quadriga.service.resolver.impl.
+     * IProjectHandleResolverManager#getProjectHandleResolvers(java.lang.String)
      */
     @Override
     public List<IProjectHandleResolver> getProjectHandleResolvers(String username) {
         List<ProjectHandleResolverDTO> resolverDtos = resolverDao.getProjectResolversForUser(username);
-        
+
         List<IProjectHandleResolver> resolvers = new ArrayList<IProjectHandleResolver>();
         resolverDtos.forEach(dto -> resolvers.add(mapper.mapProjectHandleResolver(dto)));
-        
+
         return resolvers;
     }
+
+    @Override
+    public Status validateProjectResolverHandle(IProjectHandleResolver resolver, boolean setResolverValidation) {
+
+        String resolvedHandle = resolver.buildResolvedHandle(resolver.getHandleExample());
+        if ((resolver.getResolvedHandleExample()).equals(resolvedHandle)) {
+            setResolverStatus(resolver, setResolverValidation, Status.PASSED);
+            return Status.PASSED;
+        } else {
+            setResolverStatus(resolver, setResolverValidation, Status.FAILED);
+            return Status.FAILED;
+        }
```

Comment:
instead of calling an extra method for just one if statement (seems a little overkill), why don't you do one if statement to determine the validation status ```Status status = condition ? passed : failed```

and then one more line for setting the status if it needs to be set => 3 lines of code